app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+
+const controller = require("./controller/controller");
```

Comment:
This is a really neat way of requiring in your controllers to avoid having dozens of require statements